### PR TITLE
dnd: rework the drag_and_drop tests to plain Robot

### PR DESCRIPTION
**Pull request by Saviq**

### CODE REVIEW

File: mir-ci/mir_ci/tests/robot/suites/drag_and_drop/drag_and_drop.robot [hunk 1]
```diff
@@ -0,0 +1,107 @@
+*** Settings ***
+Resource        ${KVM_RESOURCE}
+
+Suite Setup     Suite Setup
+
+
+*** Variables ***
+${T}    ${CURDIR}
```
Comment (mattkae) [inline mir-ci/mir_ci/tests/robot/suites/drag_and_drop/drag_and_drop.robot:8]: Why are you renaming `CURDIR` to `T`? (if that's what you're doing)

**Review by mattkae — CHANGES_REQUESTED** [review]
One small question, but overall makes sense

File: mir-ci/mir_ci/tests/robot/suites/drag_and_drop/drag_and_drop.robot (hunk 1 above)
Comment (Saviq) [reply mir-ci/mir_ci/tests/robot/suites/drag_and_drop/drag_and_drop.robot:8]: Just to keep it shorter below…

Comment (mattkae) [reply mir-ci/mir_ci/tests/robot/suites/drag_and_drop/drag_and_drop.robot:8]: Ok :joy: Just wanted to check that there's not a deep, robot reason here that I'm missing.

**Review by mattkae — APPROVED** [review]
I am happy

**Pull request merged**